README.md

```diff
@@ -22,6 +22,19 @@ Please note that supporting the [C Programming
 Language](https://en.wikipedia.org/wiki/C_(programming_language)) is not a goal
 of the current project.
 
+## Supported Development Platforms
+
+ Our CI pipeline builds and tests on following platforms:
```

Comment:
Should we also denote CPU architecture?  I think these are all x86 IIUC.

I've seen comments on ARM, Android, embedded support.

Reply:
Good point, yes these are xx6. Will modify it.

Reply:
To be specific, our CI only run for x86_64, not x86 in terms of 32-bit?

Reply:
Yes correct. - I though of clarifying in terms of x86 ( Intel / PC ) vs ARM ( mobile / embedded devices ). Will make it more specific : )